roles/ffh.exitnode/templates/ferm.conf.j2

```diff
@@ -13,6 +13,12 @@
 
 domain (ip) {
   table nat {
+    chain PREROUTING {
```

Comment:
Can we (start) seperating the different functional blocks within ferm's 50-exitnode.conf? Adding more functions to this single file makes is hard to read over time.